src/styles/components/_card.scss

```diff
@@ -0,0 +1,82 @@
+.card {
+  width: 20rem;
+  margin: auto;
+  padding: 3.2rem 1.6rem 1.6rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  @include border(0.1rem, $color-card-bgc, 0.5rem);
+
+  &__image {
+    width: 16rem;
+    height: 13.4rem;
+    background-image: url('../images/imac.jpeg');
+    background-size: cover;
+    margin-bottom: 4rem;
+
+    @extend %bgc-image;
+  }
+
+  &__name,
+  &__reviews,
+  &__price.value {
+    color: $color-txt-dark;
+  }
+
+  &__reviews,
+  &__prices {
+    @extend %flex-between-center;
+  }
+
+  &__code,
+  &__prices {
+    @include font(1rem, 400, 1.4rem);
+
+    margin-bottom: 1.6rem;
+  }
+
+  &__name {
+    @include font(1.2rem, 500, 1.8rem);
+
+    margin-bottom: 0.4rem;
+  }
+
+  &__code {
+    color: $color-txt-semi-dark;
+  }
+
+  &__reviews {
+    margin-bottom: 2.4rem;
+  }
+
+  &__review {
+    @include font(1rem, 400, 1.4rem);
+  }
+
+  &__price {
+    &.text {
```

Comment:
The class `&.text` does not follow BEM naming conventions. You should use `&__text` if it's an element or `&--text` if it's a modifier. Please correct this to adhere to BEM naming conventions.